recipes/argocd.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: kbrew
+app:
+  repository:
+    name: argocd
+    url: https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+    type: raw
+  name: argocd
+  namespace: "argocd"
+  sha256:
+  version: v2.0.0
+  pre_install:
+  - steps:
+    - kubectl create namespace argocd
```

Comment:
We don't need to create namespace explicitely. Kbrew will take care of that.

Reply:
When I tried to install the recipe without writing this step then it was throwing an error of "namespace does not exist.", so it looks like there is a bug.

Reply:
Which version of helm you are using?